docs/source/tutorials/otx_cli/how_to_train/detection.rst

```diff
@@ -0,0 +1,389 @@
+
+.. code-block::
+
+                            params --learning_parameters.batch_size 4 --learning_parameters.num_iters 100 --learning_parameters.enable_early_stopping false 
+
+
+5. The training results are ``weights.pth`` and ``label_schema.json`` files located in ``outputs`` folder, training logs can be found in the ``outputs/logs`` dir.
+
+.. code-block::
+  ...
+  2023-01-10 05:40:21,520 | INFO : Update Lr patience: 3
+  2023-01-10 05:40:21,520 | INFO : Update Validation Interval: 2
+  2023-01-10 05:40:21,520 | INFO : Update Early-Stop patience: 5
+  2023-01-10 05:40:23,140 | INFO : Epoch [1][1/31]        lr: 1.333e-03, eta: 11 days, 14:44:47, time: 1.619, data_time: 0.961, memory: 4673, current_iters: 0, loss_cls: 1.1261, loss_bbox: 0.6514, loss_centerness: 0.6337, loss: 2.4112, grad_norm: 18.5789
+
+  ...
+  2023-01-10 05:52:33,985 | INFO : run task done.
+  2023-01-10 05:52:35,682 | INFO : Inference completed
+  2023-01-10 05:52:35,683 | INFO : called evaluate()
+  2023-01-10 05:52:35,907 | INFO : F-measure after evaluation: 0.5487693710118504
+  2023-01-10 05:52:35,907 | INFO : Evaluation completed
+  Performance(score: 0.5487693710118504, dashboard: (1 metric groups))
+
+After about 15 minutes of training we have the PyTorch object detection model trained with OTX, that we can use for evaliation, export, optimization and deployment. 
+
+***********
+Validation
+***********
+
+1. ``otx eval`` runs evaluation of a trained model on a particular dataset.
+
+Eval function receives test annotation information and model snapshot, trained in previous step.
+Please note, ``label_schema.json`` file contains meta-information about the dataset and it should be located in the same folder as the model snapshot.
+
+The default metric is F1 measure.
+
+2. That's how we can evaluate the snapshot in ``outputs`` folder on BCCD dataset and save results to ``outputs/performance``:
+
+.. code-block::
+
+  (detection) ...$ otx eval otx/algorithms/detection/configs/detection/mobilenetv2_atss/template.yaml
+                            --test-ann-files data/wgisd/coco_annotations/test_bbox_instances.json 
+                            --test-data-roots data/wgisd/data 
+                            --load-weights outputs/weights.pth
+                            --save-performance outputs/performance.json
+  
+
+If you created ``data.yaml`` file in previous step, you can simplify the training by passing it in ``--data`` parameter. 
+Note,  with ``data.yaml``, it runs evaluation on test JSON annotation file (not on validation JSON annotation file). 
+
+.. code-block::
+
+  (detection) ...$ otx eval otx/algorithms/detection/configs/detection/mobilenetv2_atss/template.yaml
+                            --data data.yaml 
+                            --load-weights outputs/weights.pth
+                            --save-performance outputs/performance.json
```

Comment:
If we have `data.yaml` we don't even need to specify in the CLI args.
@sungmanc Could you confirm?

Reply:
Yes, you're right, but I wanted for the tutorial to show the passing arguments explicitly for a better understanding